test/distributed/checkpoint/test_2d_fsdp_dt_checkpoint.py

```diff
@@ -171,6 +208,29 @@ def test_2d_model_state_checkpoint(self) -> None:
                     else:
                         self.assertEqual(n_p1[1], n_p2[1])
 
+        def opt_at(opt, idx):
+            return list(iter(opt.state.values()))[idx]
```

Comment:
Why do we need "iter" here?

Reply:
Thanks for catching. iter is unnecessary here.